src/drivers/rtmouse.h

```diff
@@ -0,0 +1,213 @@
+#ifndef RTMOUSE_H
```

Comment:
新しく追加されたファイルなので、ライセンスの追加を推奨します。

```suggestion
/*
 *
 * rtmouse.h
 * Raspberry Pi Mouse device driver header
 *
 * Copyright (C) 2024 RT Corporation <shop@rt-net.jp>
 *
 * This program is free software; you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation; version 2.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with this program; if not, write to the Free Software
 * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
 * MA 02110-1301, USA.
 */
 
#ifndef RTMOUSE_H
```

Reply:
提案ありがとうございます。コミットしました。